packages/web/src/components/result/addons/Pagination.js

```diff
@@ -42,6 +42,8 @@ export default function Pagination(props) {
 					className={className}
 					primary={primary}
 					key={i - 1}
+					tabIndex="0"
+					onKeyPress={event => (event.key === 'Enter' ? props.setPage(i - 1) : null)}
```

Comment:
Looks good, but we already have [this helper method](https://github.com/appbaseio/reactivecore/blob/master/src/utils/helper.js#L228-L233) that you can use here.